### PR TITLE
Fix stale token symbol

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Fixes https://github.com/jbx-protocol/juice-interface/issues/1545

## how to test 

1. Go to https://rinkeby.juicebox.money/v2/p/4447
2. Observe **Tokens** section. It should use the `GBG` token.
1. Scroll to Funding Distribution section.
1. Click on `@0x1153`
1. Observe token section. It should use the `tokens` text (not the `GBG` token).

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/hooks/SymbolOfERC20.ts [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
 import { useEffect, useState } from 'react'
+import * as constants from '@ethersproject/constants'
 
 /** Returns symbol for ERC20 token with `address`. */
-export default function useSymbolOfERC20(address: string | undefined) {
+export default function useSymbolOfERC20(tokenAddress: string | undefined) {
   const [data, setData] = useState<string>()
 
-  const contract = useErc20Contract(address)
+  const contract = useErc20Contract(tokenAddress)
```
Comment (tomquirk) [inline src/hooks/SymbolOfERC20.ts:9]: The issue was that: if `tokenAddress` was 0, `contract` wouldn't change. Thus, contract.symbol() would return the old token's symbol.

Code below ensures we account for the Zero case in this hook.

**Comment by tomquirk** [comment]
@wraeth-eth could you please review?

**Comment by tomquirk** [comment]
@johnnyd-eth could pls take over the review?

File: src/hooks/SymbolOfERC20.ts (hunk 1 above)
Comment (johnnyd-eth) [reply src/hooks/SymbolOfERC20.ts:9]: Beautiful

Comment (johnnyd-eth) [reply src/hooks/SymbolOfERC20.ts:9]: Beautiful

**Review by johnnyd-eth — APPROVED** [review]
Excellent Excellent Excellent @tomquirk LGTM

**Pull request merged**